src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverseObj = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (inverseObj[value] !== undefined) {
```

Comment:
try to use other ways to check if object value is defined - [checklist](https://github.com/mate-academy/js_inverse_robot/pull/2507) point 2

Reply:
link to checklist not working : /